components/Document/DocumentViewer.tsx

```diff
@@ -54,14 +64,18 @@ type Props = {
   hasError?: boolean;
   onClose?: Function;
   showExpandBtn?: boolean;
+  isPost?: boolean;
   expandedOnlyMode?: boolean;
+  setReferenceItemDatum?: (datum) => void;
+  referenceItemDatum: any;
 };
 
 const DocumentViewer = ({
   postHtml,
   onZoom,
   viewerWidth = config.width,
   pdfUrl,
+  isPost,
```

Comment:
Instead of including this prop, you can just use the `doc` being passed in.

`Doc` is defined as a `GenericDocument` but you can use a typescript helper function called `isPost` to identify the type of object. That way, we can reduce the number of props. It is defined in `Document/lib/types`